sdk/nodejs/provider.ts

```diff
@@ -1518,14 +1528,14 @@ export namespace admissionregistration {
       * @param args The arguments to use to populate this resource's properties.
       * @param opts A bag of options that control this resource's behavior.
       */
-      constructor(name: string, args: inputApi.admissionregistration.v1alpha1.InitializerConfiguration, opts?: pulumi.CustomResourceOptions) {
+      constructor(name: string, args?: inputApi.admissionregistration.v1alpha1.InitializerConfiguration, opts?: pulumi.CustomResourceOptions) {
           let inputs: pulumi.Inputs = {};
           inputs["apiVersion"] = "admissionregistration.k8s.io/v1alpha1";
-          inputs["initializers"] = args.initializers;
+          inputs["initializers"] = args && args.initializers || undefined;
```

Comment:
Why do we need `|| undefined` here?  If `args` is not provided it will be `undefined` and so the result of the `&&` will already be `undefined` - right?

Reply:
This is pretty much copied from the AWS provider, and I kept the `||` here to make the equational reasoning explicit, which I like.